packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -307,6 +307,20 @@ const launchRole = new iam.Role(this, 'LaunchRole', {
 portfolio.setLaunchRole(product, launchRole);
 ```
 
+You can also set the launch role to use a role of a given name that exists in the local account. This can be useful when sharing a portfolio with multiple accounts, as
+the launch role in the launch role constraint will be account-agnostic resulting in fewer resources needing to be created per shared account.
+A role with the given role name must exist in the account that created the launch constraint and the account of the user who launches a product with this launch constraint.
+
+```ts fixture=portfolio-product
+import * as iam from '@aws-cdk/aws-iam';
+
+const launchRole = new iam.Role(this, 'LaunchRole', {
+  assumedBy: new iam.ServicePrincipal('servicecatalog.amazonaws.com'),
+});
+
+portfolio.setLocalLaunchRoleName(product, 'LaunchRole');
+```
+
 See [Launch Constraint](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints-launch.html) documentation
 to understand permissions roles need.
```

Comment:
should we have a "the" in between "understand permissions"

Reply:
sure, not part of this diff but might as well throw it in